apps/osim/workflows/default.yml

```diff
@@ -4,31 +4,41 @@ description: default vulnerability analysis
 priority: 0
 conditions: []
 states:
-  - name: DRAFT
+  - name: NEW
+    description: >
+      A new ticket arrived from incoming queue without any validation
+      or analysis performed yet
     requirements: []
 
-  - name: ANALYSIS
+  - name: TRIAGE
+    description: >
+      Queue captain has taken a ticket off the incoming queue and will
+      evalute if it should be rejected or qualify for further steps in
+      the workflow
     requirements:
-      - has affects
-      - has description
-      - has source
-      - has title
+      - state equals TRIAGE # check for manual change
 
-  - name: REVIEW
+  - name: PRE_SECONDARY_ASSESSMENT
+    description: >
+      Task qualified for further work and has initial data filled, triage
+      trackers has been filled and product team has been identified
     requirements:
-      - affects resolved
-      - has cve
-      - has cwe
+      - has team
+      - has trackers
+      - has title
+      - has source
+      - has affects
+      - has impact
 
-  - name: FIX
+  - name: SECONDARY_ASSESSMENT
+    description: An analyst assigned the task to themselves.
     requirements:
-      - trackers filed
+      - has owner
 
-  # you can freely comment as you wish
-  # maybe about that this is the final state
   - name: DONE
+    description: >
+      Task is done. We commonly expect that a CVE will be 
+      published, but it is not a requirement.
     requirements:
-      # TODO embargoed is not model attribute any more but annotation
-      # so this currently errors out and we need to accout for the change
-      #- not embargoed
-      - trackers resolved
+      - state equals DONE # check for manual change
```

Comment:
The way OSIM is designed a flaw will progress in the workflow if all conditions are met (or if next state has no conditions), also `Check` does not hold any info about the workflow it is in (e.g the name of the current state using it), the way I thought to pass data to it was to have the data in the check text itself. 

If you believe it will look better I could during the `apps.osim.models.py::State::__init__`  change a word/phrase dynamically for the name of the state so we can have a more self-explanatory text in workflow (maybe `manually changed` or something else). WDYT?